server/controllers/offer-type/addOfferType.js

```diff
@@ -1,3 +1,14 @@
-module.exports = (req, res) => {
-  res.send(' add new offer type');
+const { addOfferType } = require('../../database/queries/offer-type/postOfferType.js');
+
+module.exports = (req, res, next) => {
+  const name = req.body;
```

Comment:
We need to validate the name to make sure that the client gets a proper error msg